src/js/components/projects/detail.tsx

```diff
@@ -444,6 +462,12 @@ const ProjectDetail = (
           isOpen={createOrgModalOpen}
           closeModal={closeCreateOrgModal}
         />
+        <CreateTaskModal
+          project={project}
+          isOpenOrOrgId={createTaskModalOpen}
+          playgroundOrg={playgroundOrg}
+          closeCreateModal={closeCreateModal}
```

Comment:
This `closeCreateModal` fn closes the "Create Epic" modal, not this `CreateTaskModal`.